admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1124,6 +1131,43 @@ class AdminModuleController {
       self.updateModuleVisibility();
     });
   }
+
+  updateTotalResults() {
+    const updateText = (element, value) => {
+      const explodedText = element.text().split(' ');
+      explodedText[0] = value;
+      element.text(explodedText.join(' '));
+    };
+
+    // If there are some shortlist: each shortlist count the modules on the next container.
+    const $shortLists = $('.module-short-list');
+    if ($shortLists.length > 0) {
+      $shortLists.each(function shortLists() {
+        const $this = $(this);
+        updateText(
+          $this.find('.module-search-result-wording'),
+          $this.next('.modules-list').find('.module-item').length
+        );
+      });
+
+      // If there is no shortlist: the wording directly update from the only module container.
+    } else {
+      const modulesCount = $('.modules-list').find('.module-item').length;
+      updateText($('.module-search-result-wording'), modulesCount);
+
+      const selectorToToggle = (self.currentDisplay === self.DISPLAY_LIST) ?
+                               this.addonItemListSelector :
+                               this.addonItemGridSelector;
+      $(selectorToToggle).toggle(modulesCount !== (this.modulesList.length / 2));
```

Comment:
I do not understand why we should toggle based on whether the module count is equal to half the module list length ^^. What is the meaning ?

Reply:
Don't ask me something I don't understand too :D I copy paste this part of code from the old module.js file :trollface: